include/circt/Dialect/ESI/cosim/svdpi.h

```diff
@@ -0,0 +1,380 @@
+/* -*- C -*-
+ *
+ * svdpi.h
+ *
+ * SystemVerilog Direct Programming Interface (DPI).
+ *
+ * This file contains the constant definitions, structure definitions,
+ * and routine declarations used by SystemVerilog DPI.
+ *
+ * This file is from the SystemVerilog IEEE 1800-2017 Annex I.
+ */
+
+#ifndef CIRCT_DIALECT_ESI_COSIM_SVDPI_H
+#define CIRCT_DIALECT_ESI_COSIM_SVDPI_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Define size-critical types on all OS platforms. */
+#if defined(_MSC_VER)
+typedef unsigned __int64 uint64_t;
```

Comment:
can you use the existing LLVM llvm/Support/DataTypes.h for this?

Reply:
This file is a standard DPI header. It's part of the IEEE SystemVerilog standard and included with every commercial simulator verbatim. I don't want to change it, if for no other reason than to reduce potential confusion and maintain compatibility. Also, as of now none of the ESI cosim DPI code depends on LLVM and I want to keep it that way so it is severable.

Reply:
Ok, makes sense, I'd recommend moving this to somewhere really weird, e.g. `circt/include/external/dpi` since it isn't our header.